packages/babel-preset-default/package.json

```diff
@@ -37,10 +37,10 @@
 		"@babel/runtime": "^7.16.0",
 		"@wordpress/babel-plugin-import-jsx-pragma": "file:../babel-plugin-import-jsx-pragma",
 		"@wordpress/browserslist-config": "file:../browserslist-config",
-		"@wordpress/element": "file:../element",
 		"@wordpress/warning": "file:../warning",
 		"browserslist": "^4.21.9",
-		"core-js": "^3.31.0"
+		"core-js": "^3.31.0",
+		"react": "^18.2.0"
```

Comment:
I'm not entirely sure that we need this dependency here but I just mirrored the previous `@wordpress/element` dependency.

Reply:
This dependency was added by @gziolo in #17027 and I don't understand the rationale very well. The preset doesn't use `element` or `react`. The problem was something like when a project (e.g., a block) depends on `wp-scripts`, it should install everything `wp-scripts` needs automatically, instead of the project's `package.json` having to specify additional dependencies explicitly.

Reply:
I don’t remember the exact reasoning behind it. Maybe there is still value in keeping it but as a peer dependency since it’s React now. Although it doesn’t seem to be necessary anymore for plugins when using `wp-scripts build` because it converts imports to `react` or `wp.element`. It might be different with unit tests though.